Content.Shared/_Goobstation/Changeling/ChangelingComponent.cs

```diff
@@ -103,6 +103,9 @@ public sealed partial class ChangelingComponent : Component
     public float BiomassUpdateTimer = 0f;
     public float BiomassUpdateCooldown = 60f;
 
+    [DataField, AutoNetworkedField]
```

Comment:
i do not know if AutoNetworkedField is really necessary here. i just copy pasted and its late and i didnt want to think too hard